docs/api-docs/catalog/currencies/currencies-overview.md

```diff
@@ -96,17 +96,17 @@ Additionally, changing the default currency will enable the newly assigned curre
 </div>
 </div>
 
-## How Currencies Work
+## How currencies work
 
 <a id="how-currencies-work"></a>
 
-For details on how currencies are surfaced throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Works](https://developer.bigcommerce.com/api-docs/catalog/currencies/how-currencies-work)
+For details on how to surface currencies throughout BigCommerce APIs, user interfaces, and storefront components, see [How Currencies Works](https://developer.bigcommerce.com/api-docs/catalog/currencies/how-currencies-work).
 
-## Supported Features
+## Supported features
 
 <a id="supported-features"></a>
 
-See the table below for a list of supported and un-supported features.
+See the table below for a list of supported and unsupported features.

```

Comment:
We may need to rename the header for this section. It's a bit misleading to only mention supported features when we also list the unsupported ones